src/Import/src/FMI/fmi_import_context.c

```diff
@@ -26,7 +26,7 @@
 #define MODULE "FMILIB"
 
 fmi_import_context_t* fmi_import_allocate_context(jm_callbacks* callbacks) {
-    jm_log_verbose(callbacks, MODULE, "Allocating FMIL context");
+    if (callbacks) {jm_log_verbose(callbacks, MODULE, "Allocating FMIL context");}
```

Comment:
Would crash on callbacks = NULL, despite the docstring saying it would use default callbacks

Reply:
Great catch!